Projects/Home/Sources/HomeState.swift

```diff
@@ -238,13 +246,50 @@ public final class HomeStore: LoadingStateStore<HomeState, HomeAction, HomeLoadi
                     .onValue { data in
                         if let date = data.chat.messages.first?.sentAt.localDateToIso8601Date {
                             //check if it is auto generated bot message
-                            if data.chat.messages.count == 1 && date.addingTimeInterval(2) > Date() {
+                            let onlyAutoGeneratedBotMessage =
+                                data.chat.messages.count == 1 && date.addingTimeInterval(2) > Date()
+
+                            if onlyAutoGeneratedBotMessage {
                                 callback(.value(.setChatNotification(hasNew: false)))
                             } else if self.state.latestChatTimeStamp < date {
                                 callback(.value(.setChatNotification(hasNew: true)))
                             } else {
                                 callback(.value(.setChatNotification(hasNew: false)))
                             }
+
+                            var hasReceivedMoreThanOneMessage: Bool {
+                                if onlyAutoGeneratedBotMessage {
+                                    return false
+                                } else {
+                                    return !data.chat.messages.filter({ $0.sender == .hedvig }).isEmpty
+                                }
+                            }
+
+                            let hasSentMoreThanOneMessage = !data.chat.messages.filter({ $0.sender == .member }).isEmpty
+
+                            if hasReceivedMoreThanOneMessage || hasSentMoreThanOneMessage {
```

Comment:
Maybe
callback(.value(.setHasSentOrRecievedAtLeastOneMessage(hasSent: !onlyAutoGeneratedBotMessage)))

and remove hasReceivedMoreThanOneMessage &&  hasSentMoreThanOneMessage